debbsefe/Quick-Credit
# [Feature #174155852] Implement admin verify user endpoint

**What does this PR do?**

Implement API for admin to verify user

**Description of Task to be completed?**

- Create User Controller and method for admin verify
- Have the following endpoint working: PATCH /api/v1/:email/verify
- Validate user inputs
- Write tests for the endpoint
- How should this be manually tested?

git pull
git checkout ft-admin-verify-user-174155852
npm run dev
npm run test

What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/story/show/174155852
